pumpkindb_engine/src/script/macros.rs

```diff
@@ -243,6 +243,8 @@ macro_rules! alloc_and_write {
     }};
 }
 
+// TODO: use or remove?
+#[allow(unused_macros)]
```

Comment:
these macros are in fact used in tests (hence `#[cfg(test)]` below), but I didn't have time to figure out why does it complain about these when compiling for non-test profile.